hyco-https/lib/_hyco_outgoing.js

```diff
@@ -659,19 +674,22 @@ OutgoingMessage.prototype._assignSocket = function _assignSocket() {
 
   var thisPtr = this;
   var webSocket = null;
+  // this._shouldForceRendezvous = false;
+  // clearInterval(this._forceRendezvousTimer);
+  // this._forceRendezvousTimer = null;
   if (this._rendezvousChannel) {
     // A rendezvous socket has been assigned
     webSocket = this._rendezvousChannel;
     webSocket._httpMessage = this;
     webSocket.on('close', onServerResponseClose);
   } else {
     // No socket assigned yet, check the message size to decide between the control channel or a new rendezvous
-    if (shouldCreateRendezvous(this.outputSize)) {
+    if (isBufferSizeExceeded(this.outputSize) || this._shouldForceRendezvous) {
```

Comment:
>this._shouldForceRendezvous [](start = 49, length = 27)

This is the commented out item, right? Is that a problem?

Reply:
Ooops, this should not be there

---
In reply to: [340264312](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340264312) [](ancestors = 340264312)